packages/vue-flicking/src/Flicking.ts

```diff
@@ -28,29 +27,20 @@ export default class Flicking extends Vue {
   private $_pluginsDiffer!: ListDiffer<Plugin>;
   private $_cloneCount!: number;
   private $_slotDiffer!: ListDiffer<VNode>;
-  private $_slotDiffResult: DiffResult<VNode> | undefined;
-  private $_visibleIndex!: { min: number; max: number };
 
   public mounted() {
     this.$_pluginsDiffer = new ListDiffer<Plugin>();
     this.$_cloneCount = 0;
-    this.$_visibleIndex = { min: NaN, max: NaN };
 
-    this.options.renderOnlyVisible = true;
     const options = {...this.options, ...{ renderExternal: true }};
     this.$_nativeFlicking = new NativeFlicking(this.$el as HTMLElement, options);
-    this.$_slotDiffer = new ListDiffer<VNode>(this.$slots.default, (vnode, idx) => vnode.key || idx);
+    this.$_slotDiffer = new ListDiffer<VNode>(this.$slots.default, (vnode, idx) => vnode.key!);
```

Comment:
Why index is removed?

Reply:
Because vnode's key is guaranteed.

```js
  public beforeUpdate() {
    if (!this.$slots.default) {
      return;
    }

    this.$slots.default = this.$slots.default.map((node, index) => {
      if (node.key == null) {
        node.key = `${node.tag}-${index}`;
      }
      return node;
    });
  }
```